### DISCUSSION TITLE
Error in docker compose up command on windows

### DISCUSSION THREAD

**Discussion by gdunnjr** [danny-avila/LibreChat#2612] [Troubleshooting] [answered]
I'm attempting to install for docker on windows. I cloned the repo, coped the .env file and I am now getting an error when I run docker compose up:

Error response from daemon: Head "https://ghcr.io/v2/danny-avila/librechat-rag-api-dev-lite/manifests/latest": denied: denied

**Comment by danny-avila** [comment]
Thanks for trying out the project!

I've seen something like this before, you may have an expired PAT from github: https://github.com/danny-avila/LibreChat/discussions/2521

It's a public image, try troubleshooting a manual pull: https://github.com/danny-avila/rag_api/pkgs/container/librechat-rag-api-dev-lite

**Comment by gdunnjr** [comment, marked as answer]
Actually, I resolved the issue by removing login credentials for ghcr.io by running the following command: docker logout ghcr.io

**Reply by danny-avila** [reply]
Great!

**Reply by ttuleyb** [reply]
I had the same exact issue on Mac, and this solution worked. Thanks.
